boxes/tests.py

```diff
@@ -197,6 +206,43 @@ def test_sent_boxes_list(self):
             self.assertEqual(box.status, Box.DONE)
 
 
+class BoxSubmitViewTests(TestCase):
+
+    def test_valid_owner_key(self):
+        user = create_and_login_user(self.client)
+        user.public_key = VALID_KEY
+        user.fingerprint = VALID_KEY_FINGERPRINT
+        user.save()
+        box = create_open_box(user)
+        response = self.client.get(reverse("boxes_show", args=(box.uuid,)))
+        self.assertEqual(response.template_name[0], 'boxes/box_submit.html')
+
+    def test_revoked_owner_key(self):
+        user = create_and_login_user(self.client)
+        user.public_key = REVOKED_KEY
+        user.fingerprint = REVOKED_KEY_FINGERPRINT
+        user.save()
+        box = create_open_box(user)
+        response = self.client.get(reverse("boxes_show", args=(box.uuid,)))
+        self.assertEqual(response.template_name, 'boxes/closed.html')
+
+    def test_expired_owner_key(self):
```

Comment:
@dethos This test is exactly the same as test_revoked_owner_key. Am I missing something?

Reply:
Yes, you are right! it should use `EXPIRED_KEY` and `EXPIRED_KEY_FINGERPRINT`. Will fix that.